src/Symfony/Bridge/Monolog/Processor/DebugProcessor.php

```diff
@@ -45,6 +66,15 @@ public function __invoke(array $record)
      */
     public function getLogs()
     {
+        if (func_num_args() === 1) {
```

Comment:
In Symfony code we use [Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions), so this should be `1 === func_num_args ()` instead.